libqfieldsync/offline_converter.py

```diff
@@ -84,14 +84,15 @@ class OfflineConverter(QObject):
     def __init__(
         self,
         project: QgsProject,
-        export_folder: str,
+        export_filename: str,
         area_of_interest_wkt: str,
         area_of_interest_crs: Union[str, QgsCoordinateReferenceSystem],
         attachment_dirs: List[str],
         offliner: BaseOffliner,
         export_type: ExportType = ExportType.Cable,
         create_basemap: bool = True,
         dirs_to_copy: Optional[Dict[str, bool]] = None,
+        export_title: Optional[str] = None,
```

Comment:
```suggestion
        export_title: str = "",
```
Having this will make the typing easier downwards too.